### DISCUSSION TITLE
Move model_cache location to avoid issues with Kubernetes deployments?

### DISCUSSION THREAD

**Discussion by loprioj** [blakeblackshear/frigate#9577] [General]
Due to Kubernetes locking down the /config folder because of the configmap residing there, release 0.13 throws:

[Errno 30] Read-only file system: '/config/model_cache'

I didn't want to open this as an issue because it's specific to K8s, and I believe there are a few workarounds, but just wondering if there's a chance that frigate can try to create that folder somewhere else in the filesystem.

I found a few similar issues with people using Docker Compose, but the same solution won't work with K8s.

**Comment by k1n6b0b** [comment]
~~I was checking the helm chart repo for this, i'm having the same issue and its not clear the upgrade path.  in 0.12 i mounted the db at /db/ and /config is a ro configmap~~

Theres a PR waiting for the helm chart that fixes this :D https://github.com/blakeblackshear/blakeshome-charts/pull/58

**Reply by k1n6b0b** [reply]
Different issue but could have the same resolution so linking here https://github.com/blakeblackshear/blakeshome-charts/issues/50
